src/ImageSharp/Formats/Pbm/StreamExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.IO;
+using System.Text;
+
+namespace SixLabors.ImageSharp.Formats.Pbm
+{
+    internal static class StreamExtensions
+    {
+        public static int WriteDecimal(this Stream stream, int value)
+        {
+            string str = value.ToString();
+            byte[] bytes = Encoding.ASCII.GetBytes(str);
+            stream.Write(bytes);
+            return bytes.Length;
```

Comment:
Make it allocation-free:
```suggestion
            Span<byte> buffer = stackalloc byte[128];
            Utf8Formatter.TryFormat(value, buffer, out int written);
            stream.Write(buffer);
            return written;
```

Reply:
Applied your suggestion. Thanks for pointing all these optimization out to me !